src/main/java/frc/robot/Intake.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Add&named motors in Intake.
+ */
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid piston = new Solenoid(Ports.Intake.SOLENOID);
+
+    /**
+     * Add motor inverted.
+     * Add data of is Motor inverted
+     */
+    public Intake() {
+        motor.setInverted(Ports.Intake.IS_MOTOR_INVERTED);
+    }
+
+    /**
+     * Set percentage for motor.
+     * @param power the speed times the torque.
+     */
+    public void powerWheels(double power) {
+        motor.set(ControlMode.PercentOutput, power);
+    }
+
+    /**
+     * A check to see if piston is open or close, true= open, false= close.
+     * @return
+     */
+    public boolean isPistonEngaged() {
```

Comment:
As written above, it's better to name this function `isRetracted` or something similar. That way, the function's purpose is clear. Make sure you do the renaming everywhere else appropriate in this class.
Also, the piston is a hardware component, which means it is an _implementation detail_. Thus, it shouldn't be exposed to external users of the class, such as commands. Users of the class care about _what_ it does, not _how_ it does it.
Please reach out if the explanation above is not clear 🙂